disperser/dataapi/docs/docs.go

```diff
@@ -750,6 +796,27 @@ const docTemplate = `{
         "big.Int": {
             "type": "object"
         },
+        "core.PaymentMetadata": {
+            "type": "object",
+            "properties": {
+                "account_id": {
+                    "description": "AccountID is the ETH account address for the payer",
+                    "type": "string"
+                },
+                "bin_index": {
+                    "description": "BinIndex represents the range of time at which the dispersal is made",
+                    "type": "integer"
+                },
+                "cumulative_payment": {
+                    "description": "TODO: we are thinking the contract can use uint128 for cumulative payment,\nbut the definition on v2 uses uint64. Double check with team.",
```

Comment:
Do we really want this in the swagger doc?

Reply:
It's the cost of reusing data types in "core", we otherwise have to duplicate it in "dataapi". I may do a manual fix on it to handle this as an exception.

Reply:
I'm going to leave it for now until all APIs are implemented, otherwise each time we generate docs it will involve this manual edit.